src/SAML2/XML/saml/AuthenticatingAuthority.php

```diff
@@ -16,7 +16,7 @@
  * @author Tim van Dijen, <tvdijen@gmail.com>
  * @package SimpleSAMLphp
  */
-final class AuthenticatingAuthority extends \SAML2\XML\AbstractConvertable
+final class AuthenticatingAuthority extends \SAML2\XML\AbstractXMLElement
```

Comment:
This one (and the rest in the `src/SAML2/XML/saml/` directory) should extend from `SAML2\XML\saml\AbstractSamlElement`.